cmd/flowrunner/testdata/flows/brochure_test.json

```diff
@@ -155,6 +155,7 @@
                     "type": "manual"
                 },
                 "wait": {
+                    "timeout": 600,
```

Comment:
waits now keep the timeout duration around - callers should use this to control how long they wait for - and that way it doesn't matter if there's time differences between caller and engine